BookingProject/BookingWebAPI/BookingWebApi/Program.cs

```diff
@@ -95,19 +97,28 @@
     };
 });
 
-builder.Services.AddAutoMapper(typeof(MapperProfile));
+builder.Services.AddAutoMapper(typeof(UserMapper));
 
 builder.Services.AddValidatorsFromAssemblyContaining<RegisterDtoValidator>();
 //configurations
 builder.Services.Configure<JwtSettings>(builder.Configuration.GetSection("JWT"));
+builder.Services.Configure<SqlSettings>(builder.Configuration.GetSection("SqlScripts"));
 //Services
 builder.Services.AddScoped<ITokenService, TokenService>();
 builder.Services.AddScoped<IAuthenticationService, AuthenticationService>();    
 builder.Services.AddScoped<IApartamentService, ApartamentService>();
+builder.Services.AddScoped<IApartamentStatisticService, ApartamentStatisticService>();
 
 //Repositories
 builder.Services.AddScoped<IAppUserRepository, AppUserRepository>();
-builder.Services.AddScoped<IApartamentRepository, ApartamentRepository>();
+builder.Services.AddScoped<IApartamentRepository>(provider =>
+{
+    var context = provider.GetRequiredService<ApplicationDbContext>();
+    var configuration = provider.GetRequiredService<IConfiguration>();
+    var connectionString = configuration.GetConnectionString("DefaultConnection");
+    return new ApartamentRepository(context, connectionString);
+});
```

Comment:
Rework and use connection string from db context

Reply:
Fixed